src/model/formulations/Gar1962/status.jl

```diff
@@ -35,27 +95,32 @@ function _add_status_eqs!(
     switch_off = model[:switch_off]
     switch_on = model[:switch_on]
     for t in 1:model[:instance].time
-        if !g.must_run[t]
-            # Link binary variables
-            if t == 1
-                eq_binary_link[g.name, t] = @constraint(
-                    model,
-                    is_on[g.name, t] - _is_initially_on(g) ==
-                    switch_on[g.name, t] - switch_off[g.name, t]
-                )
-            else
-                eq_binary_link[g.name, t] = @constraint(
-                    model,
-                    is_on[g.name, t] - is_on[g.name, t-1] ==
-                    switch_on[g.name, t] - switch_off[g.name, t]
-                )
-            end
-            # Cannot switch on and off at the same time
-            eq_switch_on_off[g.name, t] = @constraint(
+        if g.must_run[t]
+            continue
+        end
+
+        # Link binary variables
+        # Equation (2) in Kneuven et al. (2020), originally from Garver (1962)
+        if t == 1
+            eq_binary_link[g.name, t] = @constraint(
+                model,
+                is_on[g.name, t] - _is_initially_on(g) ==
+                switch_on[g.name, t] - switch_off[g.name, t]
+            )
+        else
+            eq_binary_link[g.name, t] = @constraint(
                 model,
-                switch_on[g.name, t] + switch_off[g.name, t] <= 1
+                is_on[g.name, t] - is_on[g.name, t-1] ==
+                switch_on[g.name, t] - switch_off[g.name, t]
             )
         end
+
+        # Cannot switch on and off at the same time
+        # amk: I am not sure this is in Kneuven et al. (2020)
+        eq_switch_on_off[g.name, t] = @constraint(
+            model,
+            switch_on[g.name, t] + switch_off[g.name, t] <= 1
+        )
```

Comment:
Just a note, but if this is not described in Gar1962, we should remove it.

Reply:
Garver has
```
  is_on[g.name, t] - is_on[g.name, t-1]
  == switch_on[g.name, t] - switch_off[g.name, t]
```

However, `switch_on[gn, t]` and `switch_off[gn, t]` should not be set to 1 at the same time, but that constraint does not enforce this. I have the feeling I added this because I ran into an issue (probably due to degeneracy) in which this additional constraint was needed, but it could also have been a precautionary measure.